pyramid_openapi3/__init__.py

```diff
@@ -200,6 +202,54 @@ def spec_view(request: Request) -> FileResponse:
     config.action(("pyramid_openapi3_spec",), register, order=PHASE0_CONFIG)
 
 
+def add_spec_view_directory(
+    config: Configurator,
+    filepath: str,
+    route: str = "/spec",
+    route_name: str = "pyramid_openapi3.spec",
+) -> None:
+    """Serve and register OpenApi 3.0 specification directory.
+
+    :param filepath: absolute/relative path to the root specification file
+    :param route: URL path where to serve specification file
+    :param route_name: Route name under which specification file will be served
+    """
+
+    def register() -> None:
```

Comment:
Can we have a check here that verifies that `add_spec_view` was not called and raises an error if it was?